### PR TITLE
Add o!BWC2021 Registrations Open newspost & update wiki article

### PR DESCRIPTION
**Pull request by zeusminus**
*The third time is the charm!* 🗺️ 

- Add osu! Beatmapping World Championship 2021 Registrations Open newspost
- Update prizes, staff listing on the wiki
- Add judging criteria and a link to the official website to the wiki

### CODE REVIEW

File: news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## Team Formation
+
+Similar to the last year's edition, teams will be formed based on the country they represent. However, this year, multiple teams can be formed to represent a particular country. If multiple teams are created within a single country, only the team with the highest score amongst them will proceed with the contest upon the first round.
+
+These teams may consist of 3-6 members, where all the members must come from the same country per their osu! profile. Each team can only be created and named by its *Team Captain*, who will act as the primary point of contact between the team and the contest staff. For Captains to form their teams, all participants must be logged in to our website beforehand. Otherwise, they will not be displayed on the *Team Creation* page.
+
+Once their Team Captain selects team members, each member will need to accept the invitation to join that team. Contest staff will later finalize the team creation once everyone accepts their invitation and the team name is approved. Before that, Captains are free to change members and the team name.
```
Comment (MilkyIQ) [inline news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open.md:21]:
```suggestion
Once their Team Captain selects team members, each member will need to accept the invitation to join that team. Contest staff will later finalise the team creation once everyone accepts their invitation and the team name is approved. Before that, Captains are free to change members and the team name.
```

File: news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## Contest Format
+
+The contest has four rounds. Unlike last year, we will be hosting an **accumulative leaderboard** to grant participating teams more chances to prove themselves.
+
+As mentioned above, only the highest-scoring team of a particular country will participate in the second round. Therefore, we will announce the results of the first round. However, the rest results will be kept confidential and will only be announced after the conclusion of the final round, during our livestream. There will be no eliminations after the first round.
```
Comment (MilkyIQ) [inline news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open.md:27]:
```suggestion
As mentioned above, only the highest-scoring team of a particular country will participate in the second round. Therefore, we will announce the results of the first round. However, the rest of the results will be kept confidential and will only be announced after the conclusion of the final round, during our livestream. There will be no eliminations after the first round.
```

File: news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Each round, teams will be awarded a certain amount of points by the judges evaluating their entries. Obviously, these hard-earned points will not go to waste! Behind the scenes, these points will be added up to one another in the leaderboard, and the grand total will determine the final placement of the respective team.
```
Comment (MilkyIQ) [inline news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open.md:29]:
```suggestion
Each round, teams will be awarded a certain amount of points by the judges evaluating their entries. Obviously, these hard-earned points will not go to waste! Behind the scenes, these points will be added up with one another in the leaderboard, and the grand total will determine the final placement of the respective team.
```

File: news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+### Judges
+
+If you'd recall, we were looking for judges, details of which can be found in [this newspost](https://osu.ppy.sh/home/news/2021-04-11-osu-beatmapping-world-championship-2021-call-for-judges). We have received dozens of applicants over the span of a few weeks, and for that, we would like to thank you for your interest! Unfortunately, we have closed the osu! mode judge applications for this edition as of today, so if you're a latecomer, try to make it earlier next year! For multi-mode judges (osu!taiko, osu!catch and osu!mania), applications are still open! If you're interested, fill out the [application form](https://forms.gle/Yu2M2ZiwhapJYc7K7 "Google Forms") right away!
```
Comment (MilkyIQ) [inline news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open.md:33]:
```suggestion
If you recall, we were previously looking for judges, details of which can be found in [this newspost](https://osu.ppy.sh/home/news/2021-04-11-osu-beatmapping-world-championship-2021-call-for-judges). We have received dozens of applicants over the span of a few weeks, and for that, we would like to thank you for your interest! Unfortunately, we have closed the osu! mode judge applications for this edition as of today, so if you're a latecomer, try to make it earlier next year! For multi-mode judges (osu!taiko, osu!catch, and osu!mania), applications are still open! If you're interested, fill out the [application form](https://forms.gle/Yu2M2ZiwhapJYc7K7 "Google Forms") right away!
```
"we were" by itself just sounded a little awkward to me

File: news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 8 months of osu!supporter tag, unique profile badge, entries fast-tracked over to the qualified section |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 6 months of osu!supporter tag, unique profile badge |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 4 months of osu!supporter tag, unique profile badge |
+| **Players' Pick Winner** | 2 months of osu!supporter tag, unique profile badge |
+
+Note that Players' Pick and top 3 prizes can be added together if a team wins both prizes!
+
+## Registration
+
+Done reading and looking for where to register? You're at the right place! If you want to take part as a contestant, all you need to do is to login our website!
```
Comment (MilkyIQ) [inline news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open.md:67]:
```suggestion
Done reading and looking for where to register? You're at the right place! If you want to take part as a contestant, all you need to do is to log in to our website!
```

**Review by MilkyIQ — COMMENTED** [review]
Some grammatical changes

File: news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+Each round, teams will be awarded a certain amount of points by the judges evaluating their entries. Obviously, these hard-earned points will not go to waste! Behind the scenes, these points will be added up with one another in the leaderboard, and the grand total will determine the final placement of the respective team.
+
+### Judges
+
+If you recall, we were previously looking for judges, details of which can be found in [this newspost](https://osu.ppy.sh/home/news/2021-04-11-osu-beatmapping-world-championship-2021-call-for-judges). We have received dozens of applicants over the span of a few weeks, and for that, we would like to thank you for your interest! Unfortunately, we have closed the osu! mode judge applications for this edition as of today, so if you're a latecomer, try to make it earlier next year! For multi-mode judges (osu!taiko, osu!catch, and osu!mania), applications are still open! If you're interested, fill out the [application form](https://forms.gle/Yu2M2ZiwhapJYc7K7 "Google Forms") right away!
```
Comment (Ephemeralis) [inline news/2021-05-02-osu-beatmapping-world-championship-2021-registrations-open.md:33]:
```suggestion
If you recall, we were previously looking for judges, details of which can be found in [this newspost](https://osu.ppy.sh/home/news/2021-04-11-osu-beatmapping-world-championship-2021-call-for-judges). We have received dozens of applicants over the span of a few weeks, and for that, we would like to thank you for your interest.

We have closed the osu! mode judge applications for this edition as of today, so if you're a latecomer, try to make it earlier next year. For multi-mode judges (osu!taiko, osu!catch, and osu!mania), applications are still open - if you're interested, fill out the [application form](https://forms.gle/Yu2M2ZiwhapJYc7K7 "Google Forms") right away!
```

**Review by Ephemeralis — CHANGES_REQUESTED** [review]
some minor wording changes, otherwise looks ok

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**